user_interface/game_ui.rb

```diff
@@ -0,0 +1,59 @@
+require_relative '../modules/game'
+require 'date'
+
+class GameUserInterface
+  FILE_LOCATION = './data/games.json'.freeze
+  attr_accessor :games
+
+  def initialize
+    @games = load
+  end
+
+  def load
+    File.size(FILE_LOCATION).zero? ? [] : JSON.parse(File.read(FILE_LOCATION))
+  end
+
+  def save
+    File.write(FILE_LOCATION, JSON.pretty_generate(@games))
+  end
```

Comment:
- [ ] it would be better if the app doesn't crash when the `json` file is missing you should  handle that case and create a json file if the file is missing 
<img width="1123" alt="Screenshot 2023-02-17 at 16 21 39" src="https://user-images.githubusercontent.com/61790587/219695670-d3828541-6525-4724-83c7-76e9bf2c1500.png">